README.md

```diff
@@ -201,12 +201,37 @@ Note that the tests are meant to be used with jq 1.7.1.
 - [ ] `@format "string"` Format string
 - [ ] `label $out | break $out` Break out
 - [ ] `include "f"`, `import "f"` Include
+- [ ] CLI options
+  - [x] `--help` / `-h`
+  - [x] `--null-input` / `-n`
+  - [ ] `--raw-input` / `-R`
+  - [x] `--slurp` / `-s`
+  - [x] `--compact-output` / `-c`
+  - [x] `--raw-output` / `-r`
+  - [x] `--raw-output0`
+  - [x] `--join-output` / `-j`
+  - [x] `--color-output` / `-C`
+  - [x] `--monochrome-output` / `-M`
+  - [ ] `-L directory`
+  - [ ] `--arg name value`
+  - [ ] `--rawfile name filename`
+  - [x] `--run-tests`
+  - [ ] `--run-tests [filename]`
+  - [x] `--`
+  - [ ] Combined short options
+  - [ ] More...
+  - Non-standard CLI options
+    - [x] `--jq`
+    - [x] `--lex`
+    - [x] `--no-builtins`
+    - [x] `--parse`
+    - [x] `--repl`
 - [x] Run jqjq with jqjq
 - [x] Bugs
 
 ### jq's test suite
 
-```
+```sh
```

Comment:
I tried to rerun this, still 307, but i did notice that something seems strange with the output when running tests. Have a look at `make test` guessing it's the switch to `--join-output` that require some change?

Reply:
I just changed `./jqjq` to replace `--join-output` with `--raw-output` for `--run-tests`.

Reply:
I get 308 on `jq.test` from jq origin/main.

Reply:
Huh made sure i had master also now, still 307, weird, some locale thing? anyways, that's for another time